### PR TITLE
[vcpkg baseline][libxdamage,libxext,libxfixes] add ports

### PR DESCRIPTION
**Pull request by Neumann-A**
from https://github.com/microsoft/vcpkg/pull/9966 (https://github.com/microsoft/vcpkg/pull/21584)

fixes baseline regression in crossbuilds of libx11

### CODE REVIEW

File: ports/libxdamage/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "libxdamage",
+  "version": "1.1.5",
+  "description": "X Damage Extension library",
+  "homepage": "https://gitlab.freedesktop.org/xorg/lib/libxdamage",
+  "license": null,
```
Comment (JonLiu1993) [inline ports/libxdamage/vcpkg.json:6]: @Neumann-A, Thanks for your pr, I investigated, shouldn't the license here be MIT? https://gitlab.freedesktop.org/xorg/lib/libxdamage/-/blob/master/COPYING

File: ports/libxext/vcpkg.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "libxext",
+  "version": "1.3.4",
+  "description": "Xlib-based library for common extensions to the X11 protocol",
+  "homepage": "https://gitlab.freedesktop.org/xorg/lib/libxext",
+  "license": null,
```
Comment (JonLiu1993) [inline ports/libxext/vcpkg.json:6]: I investigated here the license should also be MIT 
https://gitlab.freedesktop.org/xorg/lib/libxext/-/blob/master/COPYING

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/libxdamage/vcpkg.json (hunk 1 above)
Comment (Neumann-A) [reply ports/libxdamage/vcpkg.json:6]: > This project is licensed under the Other. [Learn more](http://choosealicense.com/licenses/other/)

File: ports/libxext/vcpkg.json (hunk 2 above)
Comment (Neumann-A) [reply ports/libxext/vcpkg.json:6]: > This project is licensed under the Other. [Learn more](http://choosealicense.com/licenses/other/)

**Review by JonLiu1993 — APPROVED** [review]

**Review by JackBoosY — APPROVED** [review]

**Comment by JavierMatosD** [comment]
Thank you!

**Pull request merged**